### PR TITLE
update cross reference part of metadata spec

### PR DESCRIPTION
**Pull request by chenkennt**
@vicancy @vwxyzh

### CODE REVIEW

File: Documentation/spec/metadata_format_spec.md [hunk 1]
```diff
@@ -238,7 +237,21 @@ You **SHALL NOT** define new *item* in markdown.
 
 ### 3.2 Reference Items in Markdown
 
-We introduce a new markdown syntax to represent *item* reference:
+To cross reference an *item*, you can use URL with `xref` scheme. For example, to cross reference `System.String`:
```
Comment (vicancy) [inline Documentation/spec/metadata_format_spec.md]: URL => URI?

File: Documentation/spec/metadata_format_spec.md [hunk 2]
```diff
@@ -238,7 +237,21 @@ You **SHALL NOT** define new *item* in markdown.
 
 ### 3.2 Reference Items in Markdown
 
-We introduce a new markdown syntax to represent *item* reference:
+To cross reference an *item*, you can use URL with `xref` scheme. For example, to cross reference `System.String`: 
+```markdown
+xref:System.String
```
Comment (vicancy) [inline Documentation/spec/metadata_format_spec.md]: <xref:System.String>

Comment (vicancy) [inline Documentation/spec/metadata_format_spec.md]: `<xref:System.string>`

File: Documentation/spec/metadata_format_spec.md [hunk 3]
```diff
@@ -238,7 +237,21 @@ You **SHALL NOT** define new *item* in markdown.
...
+```
+
```
Comment (vicancy) [inline Documentation/spec/metadata_format_spec.md]: remove the above 3 lines?

File: Documentation/spec/metadata_format_spec.md (hunk 1 above)
Comment (chenkennt) [inline Documentation/spec/metadata_format_spec.md]: According to <https://en.wikipedia.org/wiki/Uniform_Resource_Locator>, URL is more accurate here?

**Comment by vicancy** [comment]
:+1:

**Pull request merged**